scripts/build-docs.sh

```diff
@@ -1,45 +1,14 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
That was actually better the other way (more portable)

Reply:
```suggestion
#!/usr/bin/env bash
```
Let's revert (copy from QGEP -> adapt on QGEP ;)